rsts/deployment/deployment/multicluster.rst

```diff
@@ -159,129 +330,322 @@ Create a file named ``values-override.yaml`` and add the following config to it:
+            --attrFile workflow-ecl.yaml
 
-    flytectl update execution-cluster-label --attrFile ecl.yaml
+    3. Execute a workflow indicating project and domain:
+
+      .. prompt:: bash $
+
+         pyflyte run --remote --project team1 --domain development example.py  training_workflow \                                                          ✔ ╱ docs-development-env 
+         --hyperparameters '{"C": 0.1}'
 
 Congratulations 🎉! With this, the execution of workflows belonging to a specific
-project-domain or a single workflow will be scheduled on the target label
+project-domain or a single specific workflow will be scheduled on the target label
 cluster.
+
+Day 2 Operations
+----------------
+
+Add another Kubernetes cluster
+******************************
+
+Find in this section the necessary steps to scale out your deployment by adding one Kubernetes cluster. 
+The process can be repeated for additional clusters. 
+
+.. tabbed:: AWS
+
+   
+
+  1. Create the new cluster:
+   
+     .. prompt:: bash $
+
+        eksctl create cluster --name flyte-dataplane-2 --region <AWS-REGION-CODE>  --version 1.25 --vpc-private-subnets <subnet-ID-1>,<subnet-ID-2> --without-nodegroup
+
+  .. note:: 
+
+     This is only one of multiple ways to provision an EKS cluster. Follow your organization's policies to complete this step.
+  
+
+  2. Add a nodegroup to the cluster. Typically ``t3.xlarge`` instances provide enough resources to get started. Follow your organization's policies in this regard.
+
+  4. Create an OIDC Provider for the new cluster:
+
+     .. prompt:: bash $
+
+        eksctl utils associate-iam-oidc-provider --cluster flyte-dataplane-2 --region <AWS-REGION-CODE> --approve
+
+  5. Take note of the OIDC Provider ID:
+
+     .. prompt:: bash $
+
+        aws eks describe-cluster --region <AWS-REGION-CODE> --name flyte-dataplane-2 --query "cluster.identity.oidc.issuer" --output text
+     
+  6. Go to the **IAM** section in the **AWS Management Console** and edit the **Trust Policy** of the ``flyte-dataplane-role``
+  7. Add a new ``Principal`` with the new cluster's OIDC Provider ID. Include the ``Action`` and ``Conditions`` section:
+
+  .. code-block:: json
+
+        {
+        "Version": "2012-10-17",
+        "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Federated": "arn:aws:iam::<AWS-ACCOUNT-ID>:oidc-provider/oidc.eks.<AWS-REGION-CODE>.amazonaws.com/id/<DATAPLANE1-OIDC-PROVIDER>"
+            },
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Condition": {
+                "StringLike": {
+                    "oidc.eks.<AWS-REGION-CODE>.amazonaws.com/id/<DATAPLANE1-OIDC-PROVIDER>:aud": "sts.amazonaws.com",
+
+                    "oidc.eks.<AWS-REGION-CODE>.amazonaws.com/id/<DATAPLANE1-OIDC-PROVIDER>:sub": [
+                    "system:serviceaccount:flyte:flytepropeller",
+                    "system:serviceaccount:*:default"
```

Comment:
I'd explain here that ideally we would add only the namespaces needed, i.e. project-env, so that users can be more specific if they want